src/components/organisms/AppRouter/AdminSubrouter.tsx

```diff
@@ -50,7 +50,7 @@ export const AdminSubrouter: React.FC = () => {
 
       {/* Admin V2/3/NG */}
 
-      <Route path="/admin-ng/venue/:venueId?">
+      <Route path="/admin-ng/venue/:venueId?/:selectedTab?">
```

Comment:
You will have to do the same thing for advanced settings screen as well. To keep it consistent and allow subroute navigation, there's 90% change they are going to need it there as well.

Reply:
Done for advanced settings as well in  ba2dd1d5395275e3a9cc516552882a888407661e